cmd/main.go

```diff
@@ -116,47 +127,50 @@ func main() {
 		fmt.Fprintf(os.Stderr, "Unabled to parse z: %v", err)
 		usage()
 	}
-	x, err := strconv.ParseUint(parts[1], 10, 64)
+	x, err := strconv.ParseInt(parts[1], 10, 64)
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "Unabled to parse x: %v", err)
 		usage()
 	}
-	y, err := strconv.ParseUint(parts[2], 10, 64)
+	y, err := strconv.ParseInt(parts[2], 10, 64)
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "Unabled to parse y: %v", err)
 		usage()
 	}
-	tile := slippy.NewTile(uint(z), uint(x), uint(y))
+	tile := slippy.Tile{
+		Z: slippy.Zoom(z),
+		X: uint(x),
+		Y: uint(y),
+	}
 	fileTemplate := newOutFile(tile, *tag)
 	geo, err := readInputWKT(flag.Args()[1])
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "Unabled to parse/open `%v` : %v", os.Args, err)
 		usage()
 	}
 	ctx := context.Background()
```

Comment:
nit, but use `cmd.Context()` instead of `context.Background()`

Reply:
cmd does not exist. This is an internal tool, that is straight go, and not cobra.